README.md

```diff
@@ -27,22 +27,22 @@ You can download a ready to use Docker image from our repository:
 docker pull quay.io/3scale/apicast:v2
 ```
 
-The 3scale gateway image requires one of the two environment variables:
+The 3scale gateway image requires one of two environment variables. The first option will pull the latest gateway configuration from the 3scale API Manager. The second points to a local configuration file which has already been downloaded from 3scale:
```

Comment:
Do we want to add "3scale API Manager" to the terminology? It's the first time today that I heard this way to refer to the admin portal :)